op-chain-ops/interopgen/deploy.go

```diff
@@ -278,7 +278,8 @@ func MigrateInterop(
 		chainConfigs[i] = manage.OPChainConfig{
 			SystemConfigProxy: l2Deployment.SystemConfigProxy,
 			ProxyAdmin:        superDeployment.ProxyAdmin,
-			AbsolutePrestate:  l2Cfgs[l2ChainID].DisputeAbsolutePrestate,
+			CannonPrestate:    l2Cfgs[l2ChainID].DisputeAbsolutePrestate,
+			// TODO(#17652): Configure CannonKonaPrestate
```

Comment:
I'd be tempted to remove this todo comment and just add a specific issue pointing to here. I'm not sure if we will actually wind up updating interopgen - in theory it's part of the deprecated op-e2e framework and we should build new tests in op-acceptance.  Just depends on whether we need to leverage existing op-e2e tests for coverage or not.

It's so much easier to close out an issue than have to remember to come back and remove the TODO comment.